code/proof-of-work-example.py

```diff
@@ -35,8 +42,8 @@ def proof_of_work(header, difficulty_bits):
     nonce = 0
     hash_result = ''
 
-    # difficulty from 0 to 31 bits
-    for difficulty_bits in xrange(32):
+    # difficulty from 0 to max_difficulty bits
+    for difficulty_bits in xrange(max_difficulty):
```

Comment:
@JBaczuk 

Thank you. Could you please clarify why the suggested `max_difficulty` is defaulted to 5, rather than a value of 32, here?

Reply:
Hi, I'm sorry I have no idea, this was a long time ago! It was probably just for testing.

Reply:
Hey, ok - no worries. We'll close this for now, then.